chain/network/src/peer_manager/peer_store.rs

```diff
@@ -177,20 +177,17 @@ impl PeerStore {
         }
     }
 
-    fn find_peers<F>(&self, mut filter: F, count: u32) -> Vec<PeerInfo>
+    /// Find a random subset of peers based on filter.
```

Comment:
mention what happens if count is 0 or MAX

(btw - do we still need to keep this strange behavior for 0 ? )

Reply:
@mm-near We don't. I simplified the logic, so that neither 0 nor usize::MAX is special in any way.